src/libs/Launcher.js

```diff
@@ -471,6 +471,30 @@ export default class Launcher {
     }
   }
 
+  /**
+   * Allow client konnector to run a server version of itself. Wait for the end of the execution
+   * and the returns the result of the execution
+   *
+   * @param {Object} args - any arguments we want to pass to server job. Must be serializable as string
+   * @param {Object} [options] - options object
+   * @param {Number} [options.timeout] - timeout in ms
+   * @param {Number} [options.max_exec_count] - maximum errored execution count
+   * @returns {Promise<Object>}
+   */
+  async runServerJob(args = {}, options = {}) {
+    const { client, konnector, trigger, account } = this.getStartContext() || {}
+    const jobArgs = {
+      ...args,
+      konnector: konnector.slug,
+      account: account._id,
+      folder_to_save: trigger.message?.folder_to_save
+    }
+    const { data: job } = await client
+      .collection('io.cozy.jobs')
+      .create('konnector', jobArgs, options, true)
+    return await client.collection('io.cozy.jobs').waitFor(job.id)
```

Comment:
It implies we need a way to target the server version of the connector, right?

Reply:
Yes, and since the targeted file is not the sames for clisk konnectors (main.js) and server konnectors (index.js), it works. The next goal is to be able define a more explicit names in the manifest of the konnectors.